Running server build step results in a error

### Bug Report

<!-- Fill in the relevant information below to help triage your issue. You can find the client version when inspecting the body or you can find it in the relevant package.json file. -->

|    Q               |   A
|--------------------| ------
| Breaking change    | no
| Version            | 

#### Summary
When running the command: `npx yarn build --env=dev && node dist/index.js` (dev is my personal env)
I get the following result:
```
npx: installed 1 in 1.787s
The "path" argument must be of type string
machinelabs2\server\node_modules\yarn\bin\yarn.js
yarn run v1.5.1
$ node ./build/build.js --env=dev
cp ./src/environments/personal/environment.dev.ts ./src/environments/environment.ts
Building shared libs before building the server
Building shared libs...
npx: installed 1 in 2.066s
The "path" argument must be of type string
machinelabs2\shared\models\node_modules\typescript\bin\tsc
npx: installed 1 in 1.988s
The "path" argument must be of type string
machinelabs2\shared\core\node_modules\typescript\bin\tsc
npx: installed 1 in 1.815s
The "path" argument must be of type string
machinelabs2\shared\metrics\node_modules\typescript\bin\tsc
npx: installed 1 in 1.771s
The "path" argument must be of type string
machinelabs2\shared\supervisor\node_modules\typescript\bin\tsc
'.' is not recognized as an internal or external command,
operable program or batch file.
Error: Command failed: ./node_modules/typescript/bin/tsc
Done in 19.88s.
internal/modules/cjs/loader.js:550
    throw err;
    ^

Error: Cannot find module '\machinelabs2\server\dist\index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:695:10)
    at startup (internal/bootstrap/node.js:201:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:516:3)
```
<!-- Provide a summary describing the problem you are experiencing. -->

#### Current behavior
See the stack trace above

#### How to reproduce
I can reproduce it everytime on my Windows machine just by running the command.

#### Expected behavior

<!-- What was the expected (correct) behavior? -->

#### Please execute the following commands in your terminal and post the output here
Windows
node: v9.11.1

--- a/server/build/build.js
+++ b/server/build/build.js
@@ -20,4 +20,4 @@ if (!skipShared) {
 } else {
   console.log('--skip-shared used. Not building shared libs.');
 }
-execute('./node_modules/typescript/bin/tsc');
+execute('npx tsc');

